internal/app/tfsec/custom/custom_check.go

```diff
@@ -80,23 +80,23 @@ const RequiresPresence CheckAction = "requiresPresence"
 
 // MatchSpec specifies the checks that should be performed
 type MatchSpec struct {
-	Name            string      `json:"name,omitempty"`
-	MatchValue      interface{} `json:"value,omitempty"`
-	Action          CheckAction `json:"action,omitempty"`
-	SubMatch        *MatchSpec  `json:"subMatch,omitempty"`
-	IgnoreUndefined bool        `json:"ignoreUndefined,omitempty"`
+	Name            string      `json:"name,omitempty" yaml:"name,omitempty"`
+	MatchValue      interface{} `json:"value,omitempty" yaml:"value,omitempty"`
+	Action          CheckAction `json:"action,omitempty" yaml:"action,omitempty"`
+	SubMatch        *MatchSpec  `json:"subMatch,omitempty" yaml:"subMatch,omitempty"`
+	IgnoreUndefined bool        `json:"ignoreUndefined,omitempty" yaml:"ignoreUndefined,omitempty"`
 }
 
-// CustomCheck specifies the check definition represented in json
+// CustomCheck specifies the check definition represented in yaml
```

Comment:
nitpicking here but CustomCheck should be Check in the godoc?
```suggestion
//Check specifies the check definition represented in json/yaml
```